pkg/ebpf/event_common.go

```diff
@@ -127,6 +127,11 @@ func (c ConnectionStats) String() string {
 }
 
 // ByteKey returns a unique key for this connection represented as a byte array
+// It's as following:
+
+//    32b     16b     16b      4b      4b      4/16b       4/16b
```

Comment:
b is bits in this context right? In that case srcAddr should be 32/128

Reply:
👍